src/mono/mono/tools/offsets-tool/offsets-tool.py

```diff
@@ -130,6 +130,11 @@ def require_emscipten_path (args):
 			self.target = Target ("TARGET_X86", "", IOS_DEFINES)
 			self.target_args += ["-arch", "i386"]
 			self.target_args += ["-isysroot", args.sysroot]
+		elif "x86_64-apple-darwin10" == args.abi:
+			require_sysroot (args)
+			self.target = Target ("TARGET_AMD64", "TARGET_IOS", IOS_DEFINES)
```

Comment:
should this really contain `"TARGET_IOS"`? we don't pass it on `i386-apple-darwin10` (though that might be a bug)

Reply:
That looks like a bug. These two configurations are unused, its just there for consistency.